legend-engine-core/legend-engine-core-pure/legend-engine-pure-code-compiled-core/src/main/resources/core/pure/router/router_main.pure

```diff
@@ -59,7 +59,11 @@ function meta::pure::router::routeFunction(f:FunctionDefinition<Any>[1], exeCtx:
         {|
           let unRoutedFunction = ^$l(expressionSequence = $vs, openVariables=$a.second->keys());
           let routedFunction   = routeFunction($unRoutedFunction, getPlatformRoutingStrategy(), $exeCtx, $a.second, $extensions, $debug);
-          let vars             = $a.second->put($varName->toOne(), ^List<RoutedVariablePlaceHolder>(values=^RoutedVariablePlaceHolder(name=$varName->toOne())));
+          let deactivatedEs = $routedFunction.expressionSequence->evaluateAndDeactivate();
+          let vars             = $a.second->put($varName->toOne(), ^List<meta::pure::executionPlan::PlanVarPlaceHolder>(values=^meta::pure::executionPlan::PlanVarPlaceHolder(type = $deactivatedEs->last().genericType.rawType->toOne(), 
+                                                                                                                        name=$varName->toOne(), multiplicity = $deactivatedEs->last()->toOne().multiplicity, 
+                                                                                                                        supportsStream = $deactivatedEs.multiplicity == ZeroMany )));
```

Comment:
What about one-many multiplicity?  Should we just check if is toMany, and avoid checking the lower bound?

Also, why the multiplicity value is assigned from the last expression, but the support stream is from the multiplicity directly?  Should we dont use the same logic to access the multiplicity?

`supportsStream = $deactivatedEs.multiplicity == ZeroMany`

vs

`multiplicity = $deactivatedEs->last()->toOne().multiplicity`